resources/ca_ra_utils.py

```diff
@@ -607,6 +669,9 @@ def prepare_private_key_for_kga(
     kga_cert_chain: List[rfc9480.CMPCertificate],
     password: Optional[Union[bytes, str]] = None,
     hash_alg: str = "sha256",
+    key_save_type: Optional[str] = "raw",
```

Comment:
Didn't you define `KeySaveType` with options like `RAW`, or `SEED` before? Why not use it here?